### DISCUSSION TITLE
Player list and counter

### DISCUSSION THREAD

**Discussion by delije23** [pterodactyl/panel#2815] [Feature Requests]
Something like this [https://www.mc-market.org/resources/9834/],since developer abandoned it,and there wont be for 1.1

**Comment by YajTPG** [comment]
I'm pretty sure Pterodactyl is trying not to only specialize in only game servers, so it would be better as an external addon.

**Reply by delije23** [reply]
It would be nice to have some kind of visual info on how much players are in server or who is

**Comment by HavilahOG** [comment]
> Something like this [https://www.mc-market.org/resources/9834/],since developer abandoned it,and there wont be for 1.1

I didn’t abandon it, I moved all my resources to https://quub.io and have said I’ll be remaking what I can for 1.X when I have time

**Reply by delije23** [reply]
Good to know,sorry for false accusations!
